plugins/modules/clickhouse_info.py

```diff
@@ -109,6 +110,26 @@
   returned: success
   type: dict
   sample: { "test_cluster_two_shards": "..." }
+tables:
+  description:
+    - The content of the system.tables database with names as dictionary,
+    - and the name of the tables in this dictionary is the keys.
+  returned: success
+  type: dict
+  sample: { "system": { "settings": "..." } }
```

Comment:
```suggestion
  sample: { "system": { "settings": "..." } }
  version_added: '0.3.0'
```
not sure if it works here, let's try? if it fails, just remove it